cocos/tween/tween.ts

```diff
@@ -101,7 +102,10 @@ export class Tween<T extends object = any> {
      */
     then<U extends object = any> (other: Tween<U>): Tween<T> {
         const u = other._union(true);
-        if (u) this._actions.push(u);
+        if (u) {
+            u.setSpeed(other._timeScale);
```

Comment:
Why pass time scale to speed?

Reply:
Action.setSpeed is the old method in Action class. Just use it.
Yep, if needed, we could rename Action.setSpeed to Action.setTimeScale.